arwen/common.go

```diff
@@ -33,9 +33,12 @@ const (
 	// with its corresponding callback
 	SyncCall AsyncCallExecutionMode = iota
 
-	// AsyncBuiltinFunc indicates that the async call is a cross-shard call to a
+	// AsyncBuiltinFuncCrossShard indicates that the async call is an intra-shard built in function call
```

Comment:
Update comment.

Reply:
comment is correct

Reply:
The comment is with crossShard, but the constant with intaShard